text_extensions_for_pandas/array/test_span.py

```diff
@@ -387,6 +387,50 @@ def test_contains(self):
             one_one_2_2.contains(Span(test_text, 1, 1)), [True, False]
         )
 
+    def test_addition(self):
+        arr = self._make_spans_of_tokens()
+
+        # Test SpanArray + SpanArray
+        a = arr[:2]
+        b = arr[2:]
+        result = a + b  # ["This is", "a test"]
```

Comment:
Is this right?

Reply:
```suggestion
        result = a + b  # [[0, 9) 'This is a', [5, 14) 'is a test']
```
`a` is `[ [0, 4) 'This', [5, 7) 'is' ]` and `b` is `[ [8, 9) 'a', [10, 14) 'test']`.

The assertions 2 and 3 lines down are correct.

Reply:
Ah, just my comment was wrong, doh